prac_09/my_band.py

```diff
@@ -0,0 +1,26 @@
+"""Band example with list of musicians."""
+from prac_09.band import Band
+from prac_09.musician import Musician
+from prac_09.guitar import Guitar
+
+
+def main():
+    band = Band("Extreme")
+    nuno = Musician("Nuno Bettencourt")
+    nuno.add(Guitar("Washburn N4", 1990, 2499.95))
+    nuno.add(Guitar("Takamine acoustic", 1986, 1200.0))
+    band.add(nuno)
+    band.add(Musician("Gary Cherone"))
+    pat = Musician("Pat Badger")
+    pat.add(Guitar("Mouradian CS-74 Bass", 2009, 1500.0))
+    band.add(pat)
+    kevin = Musician("Kevin Figueiredo")
+    band.add(kevin)
+
+    print("band (str)")
+    print(band)
+    print("band.play()")
+    print(band.play())
+
+
+main()
```

Comment:
Only issue I can find is this python formatting error. No newline at end of file.